packages/ssx-sdk/src/ssx.ts

```diff
@@ -27,6 +29,7 @@ declare global {
  */
 interface SSXModuleConfig {
   storage?: boolean | { [key: string]: any };
+  credentials?: boolean;
```

Comment:
Wouldn't it be better to have it as `storage`? That way we would have already structure to support other issuers.

Reply:
What do you mean by this?

Reply:
I don't think we have stuff to configure now. Can be only a boolean for now, and we add an object configuration after if necessary.